src/components/nodes/boundaryEvent/boundaryEvent.vue

```diff
@@ -78,6 +111,12 @@ export default {
       this.shape.attr('label/text', this.node.definition.get('name'));
       this.shape.component = this;
     },
+    updateSnappingPosition(task) {
```

Comment:
Would a better name for this be `updateShapePosition()`? From looking at the code it seems that the shape is thing being mutated in this method.